Formula/harmony-one.rb

```diff
@@ -0,0 +1,113 @@
+class HarmonyOne < Formula
+  desc "Harmony Blockchain Node"
+  homepage "https://github.com/harmony-one/harmony"
+  # CONTRIBUTORS: Update the following (url, sha256)
+  url "https://github.com/harmony-one/harmony/archive/refs/tags/v4.3.0.tar.gz"
+  sha256 "d463cd60973509f2a7e906582a09a7cd21cf3638b956f164c63e0536a2063367"
+  license "LGPL-3.0-or-later"
+
+  # CONTRIBUTORS: Versions may need to be updated according to documentation at:
+  #    -  https://github.com/harmony-one/harmony
+
+  depends_on "bash"
```

Comment:
Why?

Reply:
brew installs the newest version of bash wich isnt installed on macos